pkg/config/config_test.go

```diff
@@ -14,16 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package config
+package config_test
```

Comment:
any particular reason for this?

Reply:
This test only tested the public api and I've found that when tests are in the same package there is a tendency to test internal details that don't really need testing. If for some reason a package local function needs testing and testing the public api that uses it isn't sufficient we can add another test file later if that sounds ok to you.